buildstockbatch/test/test_docker_base.py

```diff
@@ -18,6 +18,52 @@
 resources_dir = os.path.join(here, "test_inputs", "test_openstudio_buildstock", "resources")
 
 
+@docker_available
+def test_skip_baseline_sims_false(basic_residential_project_file, mocker):
```

Comment:
These could be combined into a [single parametrized test](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html) with something like:
```suggestion
@pytest.mark.parametrize("skip_sims,expected_n_sims", [(False, 10), (True, 5)])
def test_skip_baseline_sims_false(basic_residential_project_file, mocker, skip_sims, expected_n_sims):
```